README.md

```diff
@@ -131,17 +134,17 @@ There are multiple ways to set these values:
 
   ```csharp
   var config = new AirbrakeConfig {
-      ProjectId = "127348",
-      ProjectKey = "e2046ca6e4e9214b24ad252e3c99a0f6"
+      ProjectId = "113743",
+      ProjectKey = "81bbff95d52f8856c770bb39e827f3f6"
```

Comment:
I'm not sure how this change is related to the NLog integration (and all other changes related to key/id). We probably want to revert these.